index.html

```diff
@@ -12,6 +12,66 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <!--Navbar-->
+    <div class="container">
+        <div class="logo">
+          <img src="/img/karma-logo.svg" alt="logo">
+          </div>
+        <nav>
+          <ul>
+          <li><b>Meet Karma</b></li>
```

Comment:
Good semantics.
Watch your indentation. Good formatting makes codes much easier to read, and errors much easier to identify.